drivers/clock_control/clock_control_nrf.c

```diff
@@ -172,9 +178,78 @@ static void set_on_state(uint32_t *flags)
 	irq_unlock(key);
 }
 
+#if CONFIG_CLOCK_CONTROL_NRF_HF_CALIBRATION
+
+static void calibration_handler(struct k_timer *timer)
+{
+	if (nrfx_clock_hfclk_is_running()) {
```

Comment:
per-nrfx-doc this function is deprecated. Please use `nrfx_clock_is_running` with `src` argument as `high-accuracy clock` instead